WaterLog_Backend/Controllers/TankLevelsController.cs

```diff
@@ -38,6 +38,12 @@ public async Task<ActionResult<TankLevelsEntry>> Get(int id)
             return await _db.TankLevels.FindAsync(id);
         }
 
+        [HttpGet("{level}")]
+        public async Task<ActionResult<TankLevelsEntry>> GetLevel(int id)
+        {
+            return await _db.TankLevels.FindAsync(id);
```

Comment:
Quick question, if DB fails to connect what happens?

Reply:
Sorry but controller has been changed. No more Tank level.